Jenkinsfile_CNP

```diff
@@ -140,4 +140,8 @@ withPipeline(type, product, component) {
     before('dbmigrate:ithc') {
         sh("./gradlew --no-daemon --init-script init.gradle assemble")
     }
+
+//    before('smoketest:dev') {
```

Comment:
Do you need this commented out code?

Reply:
Yeah, that's to be uncommented once platops sort the access issue with the script.